src/main/java/io/fusionauth/jwt/ec/ECDSASignature.java

```diff
@@ -86,10 +86,17 @@ public byte[] derDecode(Algorithm algorithm) throws IOException {
     }
 
     int len = result.length / 2;
-    //noinspection ManualMinMaxCalculation
-    System.arraycopy(r, r.length > len ? 1 : 0, result, r.length < len ? 1 : 0, r.length > len ? len : r.length);
-    //noinspection ManualMinMaxCalculation
-    System.arraycopy(s, s.length > len ? 1 : 0, result, s.length < len ? (len + 1) : len, s.length > len ? len : s.length);
+
+    int rSrcPos = r.length > len ? (r.length - len) : 0;
```

Comment:
It can occur. I don't know if I can fully explain why, but when we extract the `r` or `s` value from the length can be greater. So for example, let's say it is `66` bytes but also contains a left pad of `1`, then it would be `67` bytes and we then need to adjust the `srcPos` by one to get the correct `66` bytes.